prow/jobs/incubator/hack-showcase/hack-showcase.yaml

```diff
@@ -1,8 +1,7 @@
 job_template: &job_template
   skip_report: false
-  run_if_changed: "^hack-showcase/"
```

Comment:
run_if_changed value is a regex which is applied against files path, so it's working recursively. Path root is your repository.